Navigator.plugins, mimeTypes, pdfViewerEnabled updates (#13605)
mdn/content: files/en-us/web/api/navigator/pdfViewerEnabled/index.md

---
title: Navigator.pdfViewerEnabled

> slug: Web/API/Navigator/pdfViewerEnabled

Reviewer: ```suggestion
slug: Web/API/navigator/pdfViewerEnabled
```

Author: Thanks very much. Feel free to self-commit this sort of thing!

Reviewer: For some reason, I can't on this one.

Author: No need now anyway as the problem was incorrect filename. FYI only though, for some reason this sometimes happens if I try and commit in the conversation tab, but then works if I do it in the Files tab.

tags:
  - pdfViewerEnabled
  - API
  - DOM
  - Property
  - Reference
browser-compat: api.Navigator.pdfViewerEnabled
---
{{APIRef("HTML DOM")}}

> The **`pdfViewerEnabled`** property of the {{domxref("Navigator")}} interface should be used to determine if the browser supports inline display of PDF files when navigating to them.

Reviewer: ```suggestion
The **`pdfViewerEnabled`** property of the {{domxref("Navigator")}} interface indicates whether the browser supports inline display of PDF files when navigating to them.
```
